rules/gandalf.js

```diff
@@ -183,7 +183,9 @@ module.exports = {
       'imports': 'never',
       'exports': 'never',
       'functions': 'never'
-    }],
+    },
```

Comment:
This config does not work, must be `'comma-dangle': ['error', 'always-multiline']`

Reply:
fixed